docs/step_report.rst

```diff
@@ -0,0 +1,54 @@
+Step report
+===========
+
```

Comment:
Add the info that the step report is a web html page

Reply:
Done